drivers/sensor/scd30/Kconfig

```diff
@@ -0,0 +1,30 @@
+
+menuconfig SCD30
+    depends on I2C
+    select SENSIRION_COMMON
+    bool "Sensirion SCD30 CO2 Sensor"
+    default false
+
+if SCD30
+
+comment "SCD30 Options"
```

Comment:
Why this ?

You are already in a submenu and can see in the top navigation bar in menuconfig under which menu you are currently.